hs_core/tests/api/native/test_core_metadata.py

```diff
@@ -1062,20 +1062,12 @@ def test_relation(self):
         self.assertRaises(Exception, lambda: resource.update_metadata_element(self.res.short_id, 'relation',
                                                                               rel_to_update.id, type='isCopiedFrom'))
 
-        # test that duplicate relation types are not allowed
-        self.assertRaises(Exception, lambda: resource.create_metadata_element(
```

Comment:
Can we have multiple relations for a resource where the type and value have the same values in each of those relation objects?